### ISSUE TITLE
ocrd-anybaseocr-deskew fails on multiple output groups

### ISSUE THREAD

**Issue by sepastian** [mjenckel/ocrd_anybaseocr#53]
The [userguide](https://ocr-d.de/en/user_guide#calling-a-single-processor) states:

> Note: For processors using multiple input-, or output groups you have to use a comma separated list.
> 
> E.g.:
> 
> ocrd-anybaseocr-crop  -I OCR-D-IMG -O OCR-D-BIN,OCR-D-IMG-BIN

Deskew fails when specifying multiple output groups, because it does not split the output groups specified at comma (",").

```
root@d9e81adad635:/data/ocrd_workspace# ocrd-anybaseocr-deskew -m mets.xml -I OCR-D-IMG-BIN -O OCR-D-DESKEW-PAGE,OCR-D-DESKEW
Using TensorFlow backend.
09:23:03.986 INFO ocrd.workspace_validator - input_file_grp=['OCR-D-IMG-BIN'] output_file_grp=['OCR-D-DESKEW-PAGE', 'OCR-D-DESKEW']
09:23:09.585 INFO OcrdAnybaseocrDeskewer - INPUT FILE 0 / P_00001-PRE0001-NSJUSTIZ-BAND3-Teil_1-3_1_Vorwort_Einleitung_Inhalt-20190129T160242r
09:23:09.646 INFO OcrdAnybaseocrDeskewer - Estimating Skew Angle
09:23:15.556 INFO OcrdAnybaseocrDeskewer - Estimating Thresholds
09:23:19.107 INFO OcrdAnybaseocrDeskewer - Rescaling
09:23:19.483 INFO ocrd.workspace - created file ID: OCR-D-DESKEW_0001, file_grp: OCR-D-DESKEW, path: OCR-D-DESKEW/OCR-D-DESKEW_0001.png
Traceback (most recent call last):
  File "/usr/bin/ocrd-anybaseocr-deskew", line 8, in <module>
    sys.exit(ocrd_anybaseocr_deskew())
  File "/usr/lib/python3.6/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.6/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.6/site-packages/ocrd_anybaseocr/cli/cli.py", line 22, in ocrd_anybaseocr_deskew
    return ocrd_cli_wrap_processor(OcrdAnybaseocrDeskewer, *args, **kwargs)    
  File "/usr/lib/python3.6/site-packages/ocrd/decorators.py", line 54, in ocrd_cli_wrap_processor
    run_processor(processorClass, ocrd_tool, mets, workspace=workspace, **kwargs)
  File "/usr/lib/python3.6/site-packages/ocrd/processor/base.py", line 57, in run_processor
    processor.process()
  File "/usr/lib/python3.6/site-packages/ocrd_anybaseocr/cli/ocrd_anybaseocr_deskew.py", line 137, in process
    content=to_xml(pcgts).encode('utf-8')
  File "/usr/lib/python3.6/site-packages/ocrd/workspace.py", line 192, in add_file
    ret = self.mets.add_file(file_grp, **kwargs)
  File "/usr/lib/python3.6/site-packages/ocrd_models/ocrd_mets.py", line 224, in add_file
    el_fileGrp = self.add_file_group(fileGrp)
  File "/usr/lib/python3.6/site-packages/ocrd_models/ocrd_mets.py", line 174, in add_file_group
    raise Exception('fileGrp must not contain commas')
Exception: fileGrp must not contain commas
```

Deskewing runs successfully when specifying a single output group. However, it complains about a missing output file group for images.

```
root@d9e81adad635:/data/ocrd_workspace# ocrd-anybaseocr-deskew -m mets.xml -I OCR-D-IMG-BIN -O OCR-D-DESKEW
Using TensorFlow backend.
09:27:39.620 INFO ocrd.workspace_validator - input_file_grp=['OCR-D-IMG-BIN'] output_file_grp=['OCR-D-DESKEW']
09:27:39.624 INFO OcrdAnybaseocrDeskewer - No output file group for images specified, falling back to 'OCR-D-IMG-DESKEW'
09:27:45.275 INFO OcrdAnybaseocrDeskewer - INPUT FILE 0 / P_00001-PRE0001-NSJUSTIZ-BAND3-Teil_1-3_1_Vorwort_Einleitung_Inhalt-20190129T160242r
09:27:45.338 INFO OcrdAnybaseocrDeskewer - Estimating Skew Angle
09:27:51.250 INFO OcrdAnybaseocrDeskewer - Estimating Thresholds
09:27:54.802 INFO OcrdAnybaseocrDeskewer - Rescaling
09:27:55.177 INFO ocrd.workspace - created file ID: OCR-D-IMG-DESKEW_0001, file_grp: OCR-D-IMG-DESKEW, path: OCR-D-IMG-DESKEW/OCR-D-IMG-DESKEW_0001.png
```

I'm running inside:

```
docker run --gpus all --rm -it -v "${PWD}/Data/:/data" ocrd/all:maximum
```

**Comment by bertsky** [comment]
similar to #39 and #44.

**Comment by kba** [comment]
@sepastian I think this particular issue should be fixed in [1.0.0](https://github.com/OCR-D/ocrd_anybaseocr/releases/v1.0.0).
